packages/lib/gpcircuits/scripts/gen-circuit-parameters.ts

```diff
@@ -0,0 +1,95 @@
+import circomkitJson from "../circomkit.json";
+import { Circomkit } from "circomkit";
+import * as fs from "fs/promises";
+import { existsSync as fsExists } from "fs";
+import * as path from "path";
+import {
+  ProtoPODGPC,
+  ProtoPODGPCParameters,
+  PROTO_POD_GPC_PUBLIC_INPUT_NAMES
+} from "../src/proto-pod-gpc";
+import { batchPromise, clearDir, maxParallelPromises } from "../src/util";
+
+// Circuit parameters used to generate artifacts.
+const CIRCUIT_PARAMETERS = [
+  [1, 1, 5],
+  [1, 5, 8],
+  [3, 10, 8]
+];
+
+const circuitDir = path.join("circuits", "main");
+
+main = async (): Promise<void> => {
```

Comment:
style nit: I find using `function` rather than lambda-style easier to read, and more common in our codebase.  Obviously lambda-style is needed for inline anonymous functions, but named top-level functions like this are clearer if they look like functions.